src/components/flex/Flex.jsx

```diff
@@ -202,7 +198,8 @@ export type FlexPropsType = {
    *            component content
    *          </Flex>
    */
-  marginLeft?: FlexMarginsType
+  marginLeft?: FlexMarginsType,
+  ...
```

Comment:
what's that?

Reply:
@PinkaminaDianePie that's single thing I would check again ☝️

Reply:
it should be in next pull request only, good catch ;)